### ISSUE TITLE
Using next-> prev->next in 'Add Existing Show' does not work correctly.

### ISSUE THREAD

**Issue by rafi-d** [pymedusa/Medusa#4107]
**Branch/Commit:** 
Ver 0.22 

**OS:** 
Win 10

**What you did:** 
Added an existing show 

**What happened:** 
1. Try to progress two "steps" (select the show path, and identify it in the indexer
2. press next
3. press prev (you want to confirm that is the exact show) . Do not change anything
4. Press next again, and submit

**What you expected:**
Medusa tries to add this same show twice (fails the second time). Using next->prev->next should always ignore the previous operation, and execute the last one.

PS/Edit:
related/similar to PR #4112 ? 

**Logs:**
```
PASTE LOGS HERE

2018-04-25 16:13:22 INFO     SHOWQUEUE-ADD :: [77b2091] Trying to clean any empty folder under F:\Downloads\Bittorrent\The 100
2018-04-25 16:13:22 WARNING  SHOWQUEUE-ADD :: [77b2091] Could not add the show The 100, as it already is in your library. Error: The show The 100 has already been added by the indexer TVDBv2. Please remove the show, before you can add it through TVDBv2.
2018-04-25 16:13:19 INFO     SHOWQUEUE-ADD :: [77b2091] TVDBv2: {u'session': <medusa.session.core.MedusaSession object at 0x049EA710>, u'cache': u'D:\\Program Files\\Medusa\\Data\\cache\\indexers\\TVDBv2', u'language': u'en', u'use_zip': True}
2018-04-25 16:13:19 INFO     SHOWQUEUE-ADD :: [77b2091] Starting to add show by ShowDir: F:\Downloads\Bittorrent\The 100
2018-04-25 16:13:17 INFO     SHOWQUEUE-ADD :: [77b2091] Cache check completed
2018-04-25 16:13:15 INFO     SHOWQUEUE-ADD :: [77b2091] Selected poster for series 268592 (score=3.42844945369, score_bay=5.85689890737, rating=6.2, votes=109, res=680x1000): http://thetvdb.com/banners/posters/268592-2.jpg
2018-04-25 16:13:15 INFO     SHOWQUEUE-ADD :: [77b2091] Selected fanart for series 268592 (score=3.82391737283, score_bay=5.64783474565, rating=6.4, votes=54, res=1920x1080): http://thetvdb.com/banners/fanart/original/268592-5.jpg
2018-04-25 16:13:10 INFO     SHOWQUEUE-ADD :: [77b2091] 268592: Updating NFOs for show with new indexer info
2018-04-25 16:13:10 INFO     SHOWQUEUE-ADD :: [77b2091] Building internal name cache for The 100
2018-04-25 16:13:10 INFO     SHOWQUEUE-ADD :: [77b2091] Finished processing 2753 scene exceptions.
2018-04-25 16:13:09 INFO     SHOWQUEUE-ADD :: [77b2091] Updating exception_cache and exception_season_cache
2018-04-25 16:13:02 INFO     SHOWQUEUE-ADD :: [77b2091] Setting all previously aired episodes to the specified status: Ignored
2018-04-25 16:13:00 INFO     SHOWQUEUE-ADD :: [77b2091] 268592: Unable to find the show in the database
2018-04-25 16:12:55 INFO     SHOWQUEUE-ADD :: [77b2091] TVDBv2: {u'session': <medusa.session.core.MedusaSession object at 0x049EA710>, u'cache': u'D:\\Program Files\\Medusa\\Data\\cache\\indexers\\TVDBv2', u'language': u'en', u'use_zip': True}
2018-04-25 16:12:55 INFO     SHOWQUEUE-ADD :: [77b2091] Starting to add show by ShowDir: F:\Downloads\Bittorrent\The 100
2

**Comment by sharkykh** [comment]
About the show getting added twice, I noticed, and yes, #4112 should fix it for the Add New Show page.
About the next/prev buttons for Add Existing Show, I'm sorry but I don't understand the issue. They seem to be working fine for me for both the Existing and New pages.

**Comment by rafi-d** [comment]
Right, started with "existing show", and occurred and "new show".  This is with latest V 0.22.  I assumed you've merged all stable fixes into it couple of days ago... Anyways, see attached test-clip:

[Video_2018-04-26_213524.zip](https://github.com/pymedusa/Medusa/files/1952520/Video_2018-04-26_213524.zip)

**Comment by sharkykh** [comment]
Okay, maybe I'm missing something but I don't see anything about the next/prev buttons not working in the video?
I also recently found some other bug with the "Add Existing Shows" page, that I'll add to #4112

**Comment by rafi-d** [comment]
I think you might be right, this double-add seems  happening regardless. If you think it will be fixed in some later hot-fix to V 0.22, it's fine too, people can live with it...

**Comment by medariox** [comment]
Should be fixed, please re-open if it's still an issue.

**Comment by rafi-d** [comment]
Yeah, it's fixed in v0.24 . Thanks!
